src/services/create-profile-form-service.ts

```diff
@@ -14,8 +16,9 @@ export interface FormValues {
   remoteOnly: boolean
   position: string
   seniority: string
-  employment: string[]
+  employment: 'FULL_TIME' | 'PART_TIME' | 'CONTRACT'
```

Comment:
**nitpick:** This should be a typeof `const` or enum to prevent duplicating this string